### PR TITLE
SUPINT-1709: grid poc

### PR DESCRIPTION
**Pull request by Gabez0r**

### CODE REVIEW

File: elements/nuxeo-grid/nuxeo-grid.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    };
+  }
+
+  _updateGrid() {
+    let style = `
+  :host {
+    display: grid;
+    grid-template-columns: ${this.cols && this.cols > 1 ? '1fr '.repeat(this.cols) : 'auto'};
+    grid-template-rows: ${this.rows && this.rows > 1 ? 'auto '.repeat(this.rows) : 'auto'};
+    ${this.gap ? `grid-gap: ${this.gap}` : ''};
```
Comment (richardsd) [inline elements/nuxeo-grid/nuxeo-grid.js:140]: How does it work if we specify both the `gap` and `columnGap` or `rowGap`?

Comment (Gabez0r) [reply elements/nuxeo-grid/nuxeo-grid.js:140]: We just let the browser decide. Usually, column and row gap have precedence.

File: elements/nuxeo-grid/nuxeo-grid.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+    ${this.columnGap ? `grid-column-gap: ${this.columnGap};` : ''}
+    ${this.rowGap ? `grid-row-gap: ${this.rowGap};` : ''}
+    ${this.alignItems ? `align-items: ${this.alignItems};` : ''}
+    ${this.justifyItems ? `justify-items: ${this.justifyItems};` : ''}
+  }
+
+    `;
+    Array.from(this.children)
```
Comment (richardsd) [inline elements/nuxeo-grid/nuxeo-grid.js:148]: Is this a polymer helper to get the children elements in the DOM? Never saw it

Comment (Gabez0r) [reply elements/nuxeo-grid/nuxeo-grid.js:148]: It's part of the DOM api. [Parent nodes expose this property to get the children](https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/children).

Comment (richardsd) [reply elements/nuxeo-grid/nuxeo-grid.js:148]: ah, my bad (stupidity should say) 🤦 `this` is a DOM node by itself

File: elements/nuxeo-grid/nuxeo-grid.js (hunk 1 above)
Comment (richardsd) [reply elements/nuxeo-grid/nuxeo-grid.js:140]: ok 👍

**Pull request closed without merging**